docs/development/getting-started.rst

```diff
@@ -178,14 +179,38 @@ application.
    <https://www.elastic.co/guide/en/elasticsearch/reference/6.2/disk-allocator.html>`_.
 
 
-In a terminal run the command:
+Once ``make build`` has finished,  run the command:
 
 .. code-block:: console
 
     make serve
 
-This command will produce output for a while, and will not exit. While it runs,
-open a second terminal, and run:
+This command starts the containers that run Warehouse on your local machine.
+After the initial build process, you will only need this command each time you
+want to run Warehouse locally.
```

Comment:
Maybe change "run" to "startup" to indicate that this should be done if the user ever stops the continuously running `make serve` command?